pyfstat/make_sfts.py

```diff
@@ -370,9 +373,33 @@ def run_makefakedata(self):
         cl_mfd.append("--outSingleSFT=TRUE")
         cl_mfd.append('--outSFTdir="{}"'.format(self.outdir))
         cl_mfd.append('--outLabel="{}"'.format(self.label))
-        cl_mfd.append("--IFOs={}".format(self.IFOs))
+
         if self.add_noise:
+
```

Comment:
could you remove this blank line and 389 too so the block hierarchy stays cleaner